covid-nlp/main.ipynb

```diff
@@ -0,0 +1,2384 @@
+{
```

Comment:
<p> I talked to benny about how to solve the issue with text replacement being not idiomatic of the theoretic framework.</p><p>We managed to converge on the following simple solution.</p><p> </p><p>We simply preprocess the original documents by lemmatizing and adding POS annotations at the beggining imperatively,</p><p>The we do all of our regexes over POS annotated and lemmatized documents,</p><p>So instead of:</p><p> </p><p><code>Corona is Positive</code> we will have <code>Corona&lt;Noun&gt; be&lt;Verb&gt; Positive&lt;SomePosTag&gt;</code> .</p><p> </p><p>What do you think about this?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Please address this comment.</p>

Reply:
No no, i meant to modify the text to only lematize the words you care about, but just to do it in python before any rgxlog code, and then have the rgxlog code's input be the lematized document.

There is no need to use spans.